internal/pkg/deploy/cloudformation/stack/transformers.go

```
@@ -460,6 +481,132 @@ func convertEnvSecurityGroupCfg(mft *manifest.Environment) (*template.SecurityGr
 	}, nil
 }
 
+func (s *LoadBalancedWebService) convertApplicationLoadBalancer() (*template.ALBListener, error) {
+	albConfig := s.manifest.RoutingRule
+	if albConfig.Disabled() || albConfig.IsEmpty() {
+		return nil, nil
+	}
+	var rules []template.ALBListenerRule
+	// build listener rule config from primary rule config from manifest.
+	rule, err := RoutingRuleConfigOrBoolConverter{
+		rule:         albConfig,
+		manifest:     s.manifest,
+		httpsEnabled: s.httpsEnabled,
+	}.convert()
+	if err != nil {
+		return nil, nil
+	}
```

Should we return the error?